TheSufferfest/node-fit
# WB-78 WB-71 WB-72 - Create fit file from activity data generated by SufferFestWeb,  Upgrade SDK, export data to NodeJS

#### What’s this PR do?
Create a .fit file base on a json object 
#### Any background context you want to provide?
- This update change the json structure base on the data available in SufferFestWeb
- The activity-from-data.js was generated with data from a real activity by SufferFestWeb
#### How should this be manually tested?

1. `npm install`
2. `npm test`
3. It should pass the test
4. Go to examples\content-to-js
5. `npm install`
6. `node index.js`
7. Verify that a file called temp.fit
8. Open the fit file in a desktop application like Golden Cheetah and review the content. (This file is created from NodeJS)

#### Could any automated tests be added for this?
Maybe
#### What are the relevant JIRA issues?
https://thesufferfest.atlassian.net/browse/WB-70
#### Screenshots (if appropriate)
#### Open issues
